arlas-admin/src/main/java/io/arlas/server/task/CollectionAutoDiscover.java

```diff
@@ -61,7 +62,7 @@ public void execute(ImmutableMultimap<String, String> arg0, PrintWriter arg1) th
             List<CollectionReferenceDescription> discoveredCollections = admin.getAllIndecesAsCollections();
```

Comment:
Should we not correct Indeces -> Indices ?

Reply:
For a reason I don't understand, there are 16 repetitions of "indeces" in the code, I think it should rather be part of a dedicated PR

Reply:
ok for me!